filters/apiusagemonitoring/spec.go

```diff
@@ -77,7 +78,7 @@ func NewApiUsageMonitoring(
 	}
 	unknownPath := newPathInfo(
 		unknownPlaceholder,
-		unknownPlaceholder,
+		noTagPlaceholder,
```

Comment:
@tkrop the rationale for using `{unknown}` instead of `{no-tag}`: We don't know what the right tag is, so saying it has no tag is more than we know.

Reply:
Okay. Feel free to role this back. I did this because of an request of @aryszka.

Reply:
Also fine for me.